app/Http/Controllers/UserController.php

```diff
@@ -39,10 +41,14 @@ public function editProfile($id)
     public function update(Request $request, $id)
     {
         $user = User::find($id);
-        $user->name = $request->input('name');
+        // dd($request->all());
```

Comment:
Please remove this dd(), this should only be here when you are debugging and should be removed when you create a PR :)